test-override/jenkins-params-default.include.sh

```diff
@@ -22,10 +22,11 @@ echo '
 # Config override script to exclude default notebooks.
 
 # Add more -e for additional nbs to blacklist.
-#NEW_NB_LIST="$(echo $NOTEBOOKS | sed \
-#  -e "s@$RAVENPY_DIR/docs/notebooks/HydroShare_integration.ipynb@@" \
-#  )"
-#NOTEBOOKS="$NEW_NB_LIST"
+# esgf-dap: https://github.com/Ouranosinc/pavics-sdi/issues/353
+NEW_NB_LIST="$(echo $NOTEBOOKS | sed \
+  -e "s@$PAVICS_SDI_DIR/docs/source/notebooks/esgf-dap.ipynb@@" \
+  )"
+NOTEBOOKS="$NEW_NB_LIST"
```

Comment:
What about having a `BLACKLIST_NOTEBOOKS` variable for defining a list of notebooks that are fed to some logic that ensures that the final `NOTEBOOKS` list doesn't include them? `NEW_NB_LIST` is confusing.

Reply:
Finally we do not need to blacklist anymore since the nb restart to work with PR https://github.com/Ouranosinc/pavics-sdi/pull/355.  But we can still merge this PR for the nicer blacklist mechanism.